src/_includes/layouts/home.liquid

```diff
@@ -10,9 +10,9 @@ layout: layouts/base.liquid
       </div>
     </div>
   </div>
-  <div class="bg-illustration container w-full px-[0.9375rem] mx-auto pt-12 pb-32" style="background-image: url('/assets/img/illustration-gov.png');">
+  <div class="bg-illustration container w-full px-[0.9375rem] mx-auto pt-12 desktop:pb-32" style="background-image: url('/assets/img/illustration-gov.png');">
```

Comment:
@SeaJerry Not a big deal but generally, we want to use the tailwind default breakpoints when we can so that it doesn't force archeology to figure out where `desktop` or `tablet` came from.